src/components/Rockets.js

```diff
@@ -14,7 +16,27 @@ const Rockets = () => {
   }, []);
 
   return (
-    <div className="profile">coming soon</div>
+    <div className="">
+      {rockets.map((rocket) => (
+        <Card
+          bg="ligh"
+          key={rocket.id}
+          text="dark"
+          className="m-2 row flex-row align-items-center p-4"
+        >
+          <div className="col-sm-12 col-md-3 text-center ">
+            <Card.Img variant="left" src={rocket.flickr_images[0]} className="w-100 rocket-image" height="200" style={{ maxWidth: '25vw', maxHeight: '25vw' }} />
```

Comment:
```suggestion
            <Card.Img variant="left" src={rocket.flickr_images[0]} style={{ width: '250px', height: '250px', objectFit: 'cover' }} />
```
Nice work, @Emmanuelaaron! What do you think about this suggestion?

Reply:
Thanks, @enionsouza for taking note of the image rendering. I'm implementing the suggested updates